src/c-writer.cc

```diff
@@ -1057,22 +1059,27 @@ void CWriter::Write(const StackVar& sv) {
   }
 }
 
-void CWriter::Write(Type type) {
+// static
+const char* CWriter::GetCTypeName(const Type& type) {
```

Comment:
Unrelated to this PR but what do you think about making these helpers into normal static function rather than method?

It would avoid the awkard `// static` comments and improve locality of reference (there really is no need to declare this function up on ling 265 right?

Maybe as a followup we could convert them all.